src/test/java/uk/gov/hmcts/reform/professionalapi/service/impl/JurisdictionServiceImplTest.java

```diff
@@ -78,7 +78,10 @@ public void should_create_jurisdictionuserprofilerequest_for_super_user() {
 
         JurisdictionUserCreationRequest request = jurisdictionServiceImpl.createJurisdictionUserProfileRequestForSuperUser(user);
         assertThat(request).isNotNull();
+        assertThat(request.getId()).isNotEmpty();
```

Comment:
is this test case  is required for nightly build to pass?

Reply:
yeah the mutation score was below threshold

Reply:
ok.